### DISCUSSION TITLE
workflow into RPA flow

### DISCUSSION THREAD

**Discussion by a7medo778** [Skyvern-AI/skyvern#1140] [Ideas]
hey guys,
so while the browser llm is great for discovery, its painfully flow.

so what if once the navigation and flow is verified and confirmed, it can be turned into an RPA flow that can scrap/automate actions much faster

**Comment by suchintan** [comment]
This is a great idea. We have a beta version of it available under workflows (you can cache individual blocks) -- keep your eyes peeled for more improvements coming!
